code/_core/mob/living/advanced/player/_player.dm

```diff
@@ -116,17 +116,9 @@
 	expiration_time = SECONDS_TO_DECISECONDS(180)
 
 /mob/living/advanced/player/get_examine_list(var/mob/living/examiner)
-	var/list/species_to_names = list(
-		"human" = "human",
-		"reptile_advanced" = prob(10) ? "spine dragger" : "lizard",
-		"cyborg" = prob(10) ? "tin can": "cyborg",
-		"diona" = "dionae",
-		"moth" = prob(10) ? "moff" : "moth"
-	)
-
 	. = list(
 		div("examine_title", src.name),
-		div("center bold","Level [level] [species_to_names[species]]"),
+		div("center bold","Level [level] [SPECIES(species)]"),
```

Comment:
SPECIES(species) gets the /datum/, not the name of it. I should've been clearer so that's my bad.

Reply:
when you display datums, it shows names. So not really a big deal, works

Reply:
Yeah displaying a reference to something in the chat just shows its name. Want it to be changed to species.name for clarification or leave it as-is? There's no real difference

Reply:
Should be species.name because datums don't usually have names, but somehow it works like that. A future update could mess it up so it should be species.name.